tests/storage/hpp/test_hostpath.py

```diff
@@ -32,6 +32,7 @@
 
 import tests.storage.utils as storage_utils
 from tests.storage.constants import HPP_STORAGE_CLASSES
+from tests.storage.golden_image.test_golden_image import LATEST_RHEL_IMAGE
```

Comment:
We should not import from the `test_` files. We should store constants in the `constants.py` file - the closest to where it's used.
(but here you may not even need this constant, please see my next comment)